espanol/homero/homero.txt

```diff
@@ -0,0 +1 @@
+"Dame ese bate, dame ese baaate, dame ese bate". De repente, alguien le pone un espejo en frente, el se asusta y cae por las escaleras.
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.